server/src/handlers/http/logstream.rs

```diff
@@ -266,8 +266,16 @@ pub async fn get_stats(req: HttpRequest) -> Result<impl Responder, StreamError>
 
     let time = Utc::now();
 
+    let stream_init_time = &CONFIG
+        .storage()
+        .get_object_store()
+        .get_whole_json(&stream_name)
+        .await
+        .unwrap()["created-at"];
+
```

Comment:
We shouldn't have to get this from object store every time. There is an in-memory map maintained at Parseable server to avoid this exact same situation. Please use the map.

Reply:
resolved it in the latest commit .... now the created at data is loaded from the file either on load or once the file is created